### ISSUE TITLE
运行topformer出现好多warnning

### ISSUE THREAD

**Issue by loxoo6** [PaddlePaddle/PaddleSeg#3384]
### 问题确认 Search before asking

- [X] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleSeg/issues)(包括open与closed)，没有发现相似的bug。I have searched the [open and closed issues](https://github.com/PaddlePaddle/PaddleSeg/issues) and found no similar bug report.

### Bug描述 Describe the Bug

2023-07-18 09:20:02 [INFO]	
------------Environment Information-------------
platform: Linux-4.15.0-140-generic-x86_64-with-debian-stretch-sid
Python: 3.7.4 (default, Aug 13 2019, 20:35:49) [GCC 7.3.0]
Paddle compiled with cuda: True
NVCC: Build cuda_11.2.r11.2/compiler.29618528_0
cudnn: 8.2
GPUs used: 1
CUDA_VISIBLE_DEVICES: None
GPU: ['GPU 0: Tesla V100-SXM2-32GB']
GCC: gcc (Ubuntu 7.5.0-3ubuntu1~16.04) 7.5.0
PaddleSeg: 2.7.0
PaddlePaddle: 2.3.2
OpenCV: 4.1.1
------------------------------------------------
2023-07-18 09:20:02 [INFO]	
---------------Config Information---------------
batch_size: 4
export:
  transforms:
  - keep_ratio: true
    size_divisor: 32
    target_size:
    - 2048
    - 512
    type: Resize
  - mean:
    - 0.485
    - 0.456
    - 0.406
    std:
    - 0.229
    - 0.224
    - 0.225
    type: Normalize
iters: 160000
loss:
  coef:
  - 1
  types:
  - ignore_index: 255
    type: OhemCrossEntropyLoss
lr_scheduler:
  end_lr: 0
  learning_rate: 0.0012
  power: 1.0
  type: PolynomialDecay
  warmup_iters: 1500
  warmup_start_lr: 1.0e-06
model:
  backbone:
    in_channels: 3
    lr_mult: 0.1
    pretrained: https://paddleseg.bj.bcebos.com/dygraph/backbone/topformer_base_imagenet_pretrained.zip
    type: TopTransformer_Base
  num_classes: 3
  type: TopFormer
optimizer:
  type: AdamW
  weight_decay: 0.01
train_dataset:
  dataset_root: /home/aistudio/PaddleSeg/data
  img_channels: 3
  mode: train
  num_classes: 3
  train_path: /home/aistudio/PaddleSeg/data/train_list.txt
  transforms:
  - max_scale_factor: 2.0
    min_scale_factor: 0.5
    scale_step_size: 0.25
    type: ResizeStepScaling
  - crop_size:
    - 512
    - 512
    type: RandomPaddingCrop
  - type: RandomHorizontalFlip
  - brightness_range: 0.4
    contrast_range: 0.4
    saturation_range: 0.4
    type: RandomDistort
  - mean:
    - 0.485
    - 0.456
    - 0.406
    std:
    - 0.229
    - 0.224
    - 0.225
    type: Normalize
  type: Dataset
val_dataset:
  dataset_root: /home/aistudio/PaddleSeg/data
  img_channels: 3
  mode: val
  num_classes: 3
  transforms:
  - keep_ratio: true
    size_divisor: 32
    target_size:
    - 2048
    - 512
    type: Resize
  - mean:
    - 0.485
    - 0.456
    - 0.406
    std:
    - 0.229
    - 0.224
    - 0.225
    type: Normalize
  type: Dataset
  val_path: /home/aistudio/PaddleSeg/data/val_list.txt
------------------------------------------------
W0718 09:20:02.737715  4063 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.0, Driver API Version: 11.2, Runtime API Version: 11.2
W0718 09:20:02.737746  4063 gpu_resources.cc:91] device: 0, cuDNN Version: 8.2.
2023-07-18 09:20:03 [INFO]	Loading pretrained model from https://paddleseg.bj.bcebos.com/dygraph/backbone/topformer_base_imagenet_pretrained.zip
Connecting to https://paddleseg.bj.bcebos.com/dygraph/backbone/topformer_base_imagenet_pretrained.zip
Downloading topformer_base_imagenet_pretrained.zip
[==================================================] 100.00%
Uncompress topformer_base_imagenet_pretrained.zip
[==================================================] 100.00%
2023-07-18 09:20:04 [WARNING]	SIM.1.local_embedding.conv.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.local_embedding.bn.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.local_embedding.bn.bias is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.local_embedding.bn._mean is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.local_embedding.bn._variance is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_embedding.conv.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_embedding.bn.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_embedding.bn.bias is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_embedding.bn._mean is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_embedding.bn._variance is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_act.conv.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_act.bn.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_act.bn.bias is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_act.bn._mean is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.1.global_act.bn._variance is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.local_embedding.conv.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.local_embedding.bn.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.local_embedding.bn.bias is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.local_embedding.bn._mean is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.local_embedding.bn._variance is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_embedding.conv.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_embedding.bn.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_embedding.bn.bias is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_embedding.bn._mean is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_embedding.bn._variance is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_act.conv.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_act.bn.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_act.bn.bias is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_act.bn._mean is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.2.global_act.bn._variance is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.local_embedding.conv.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.local_embedding.bn.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.local_embedding.bn.bias is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.local_embedding.bn._mean is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.local_embedding.bn._variance is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_embedding.conv.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_embedding.bn.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_embedding.bn.bias is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_embedding.bn._mean is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_embedding.bn._variance is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_act.conv.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_act.bn.weight is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_act.bn.bias is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_act.bn._mean is not in pretrained model
2023-07-18 09:20:04 [WARNING]	SIM.3.global_act.bn._variance is not in pretrained model
2023-07-18 09:20:04 [INFO]	There are 278/323 variables loaded into TopTransformer.
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/paddle/nn/layer/norm.py:654: UserWarning: When training, we now always track global mean and variance.
  "When training, we now always track global mean and variance.")
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/paddle/fluid/dygraph/math_op_patch.py:278: UserWarning: The dtype of left and right variables are not the same, left dtype is paddle.float32, but right dtype is paddle.int64, the right dtype will convert to paddle.float32
  format(lhs_dtype, rhs_dtype, lhs_dtype))
2023-07-18 09:20:20 [INFO]	[TRAIN] epoch: 1, iter: 10/160000, loss: 1.3124, lr: 0.000008, batch_cost: 1.6149, reader_cost: 1.22058, ips: 2.4770 samples/sec | ETA 71:46:05

### 复现环境 Environment

paddlepaddle 2.3.2
paddleseg 2.7.0
平台 aistudio

### Bug描述确认 Bug description confirmation

- [X] 我确认已经提供了Bug复现步骤、代码改动说明、以及环境信息，确认问题是可以复现的。I confirm that the bug replication steps, code change instructions, and environment information have been provided, and the problem can be reproduced.

### 是否愿意提交PR？ Are you willing to submit a PR?

- [X] 我愿意提交PR！I'd like to help by submitting a PR!

**Comment by Asthestarsfalll** [comment]
加载预训练模型时缺少权重的警告属于正常情况。

**Comment by ToddBear** [comment]
以上回答已经充分解答了问题，如果有新的问题欢迎随时提交issue，或者在此条issue下继续回复～
我们开启了飞桨套件的ISSUE攻关活动，欢迎感兴趣的开发者参加：https://github.com/PaddlePaddle/PaddleOCR/issues/10223
